### ISSUE TITLE
AMD Architecture?

### ISSUE THREAD

**Issue by AJ** [chadgeary/cloudblock#109]
Hello Chad,

One of the commands in the readme file is 
deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main

I have an intel processor. Is the architecture string okay for an intel processor?

**Comment by chadgeary** [comment]
yes, see https://en.wikipedia.org/wiki/X86-64 for more information about architectures.
